lib/Jsrt/Jsrt.cpp

```diff
@@ -869,10 +869,14 @@ CHAKRA_API JsSetContextData(_In_ JsContextRef context, _In_ void *data)
 void HandleScriptCompileError(Js::ScriptContext * scriptContext, CompileScriptException * se, const WCHAR * sourceUrl)
 {
     HRESULT hr = se->ei.scode;
-    if (hr == E_OUTOFMEMORY || hr == VBSERR_OutOfMemory || hr == VBSERR_OutOfStack || hr == ERRnoMemory)
+    if (hr == E_OUTOFMEMORY || hr == VBSERR_OutOfMemory || hr == ERRnoMemory)
```

Comment:
Review: Please help check if removing `hr == VBSERR_OutOfStack` here is correct. I removed it because otherwise normal out of stack becomes fatal OOM too. That is inconsistent to how we handle normal out of stack in runtime.

Reply:
I agree with you. The vbserr_outofstack should not throw OOM exception.